src/components/diagram-3b-table.js

```diff
@@ -1,10 +1,10 @@
 import React from "react"
 
 const Table3b = () => (
-  <section className="cmp-diagram-table">
+  <section className="cmp-diagram">
     <p id="table3b" className="font-diagram-heading">Q: What about a client engagement leads you to suggest or not suggest a design system?</p>
     <p className="font-diagram-copy">79 Agency Responses | Select all that apply</p>
-    <div className="cmp-diagram-table__scrollable">
+    <div className="cmp-diagram__table">
```

Comment:
Is the `-diagram` part confusing on these? Should the class names be something more like `cmp-table` and `cmp-chart`? Maybe we don't need `diagram` at all 🤔

Reply:
Technically `diagram` is a container that holds a table and chart so changing it to `table` would make it redundant. This all might change with the IE11 fix too

Reply:
Got it, good point 👍